PENDING.md

```diff
@@ -59,6 +59,7 @@ IMPROVEMENTS
 * SDK
     * [tools] Make get_vendor_deps deletes `.vendor-new` directories, in case scratch files are present.
     * [cli] \#1632 Add integration tests to ensure `basecoind init && basecoind` start sequences run successfully for both `democoin` and `basecoin` examples.
+    * [store] Speedup IAVL iteration, and consequently everything that requires IAVL iteration. [#2143](https://github.com/cosmos/cosmos-sdk/issues/2143)
```

Comment:
Mhmmm, I think it's finally worthwhile committing to a change log format standard.

Reply:
Can we not block this pr on that tho, and decide it in a  separate pr and format accordingly there.

Reply:
I'm not suggesting we block (I approved actually). I'm just stating it here for reference

Reply:
Thanks! I misunderstood, totally agree with standardizing this